service/scaladsl/server/src/main/scala/com/lightbend/lagom/scaladsl/server/LagomApplicationLoader.scala

```diff
@@ -245,6 +245,7 @@ abstract class LagomApplication(context: LagomApplicationContext)
     applicationLifecycle.addStopHook(stopHook)
     system
   }
+  override lazy val coordinatedShutdown: CoordinatedShutdown = CoordinatedShutdown(actorSystem)
```

Comment:
This causes the test failure, because it takes precedence over the definition in Play's `BuiltInComponents`, which uses `CoordinatedShutdownProvider` and therefore registers a shutdown task to stop `applicationLifecycle`.

This version does not stop the `applicationLifecycle`. It looks like we can simply remove it.